exercises/H-array-literals/exercise.js

```diff
@@ -4,8 +4,8 @@
   Declare some variables assigned to arrays of values
 */
 
-let numbers = []; // add numbers from 1 to 10 into this array
-let mentors; // Create an array with the names of the mentors: Daniel, Irina and Rares
+let numbers = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]; // add numbers from 1 to 10 into this array
```

Comment:
Well done Selchuk for a great job. However, I read here [https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas] the importance of 'trailing commas'. You can have a read and see if something you might want to consider doing.

Reply:
Thank you for sharing the link. It sounds like an interesting topic. I'll take a look at it later.